<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="main_frame_layout">
       <item>
        <widget class="QLineEdit" name="translate_le">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="spanish_btn_layout">
         <item>
          <widget class="QPushButton" name="spanish_line_back_btn">
           <property name="text">
            <string>&lt; Sentence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spanish_line_forward_btn">
           <property name="text">
            <string>Sentence &gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="english_le"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="englis_btn_layout">
         <item>
          <widget class="QPushButton" name="english_line_back_btn">
           <property name="text">
            <string>&lt; Sentence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="english_line_forward_btn">
           <property name="text">
            <string>Sentence &gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="btn_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="chapter_back_btn">
            <property name="text">
             <string>&lt; Chapter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chapter_forward_btn">
            <property name="text">
             <string>Chapter &gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sentence_back_btn">
            <property name="text">
             <string>&lt; Sentence</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sentence_forward_btn">
            <property name="text">
             <string>Sentence &gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
